dashboard/test/ui/features/foundations/footer.feature

```diff
@@ -10,29 +10,10 @@ Feature: Checking the footer appearance
 
     When I press the first ".copyright-link" element
     And I wait for 0.25 seconds
-    Then I see no difference for "copyright flyout"
-    And I press the first ".copyright-link" element
+    Then I see no difference for "copyright dialog"
+    And I press the first "#ui-close-dialog" element
     And I wait for 0.25 seconds
 
-    When I drag the visualization grippy by -400 pixels
-    Then I see no difference for "smaller small footer"
-    And I wait for 0.25 seconds
-
-    When I press the first ".copyright-link" element
-    And I wait for 0.25 seconds
-    Then I see no difference for "smaller copyright flyout"
-    And I press the first ".copyright-link" element
-    And I wait for 0.25 seconds
-
-    # Now, variations where we resize while the flyouts are open, to make
-    # sure they update their size/position properly during the resize
-
-    When I press the first ".copyright-link" element
-    And I wait for 0.25 seconds
-    Then I see no difference for "copyright resize before"
-    When I drag the visualization grippy by -400 pixels
-    Then I see no difference for "copyright resize after"
-
```

Comment:
The copyright is not a flyout anymore, it's a Dialog, therefore we don't need to test that the flyout updates its size/position.